tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1277,3 +1277,109 @@ Events:
 		This event forwards data received over L2CAP channel.
 		Chan_ID is the internal application number that identifies
 		L2CAP channel.
+
+Mesh Node Service (ID 4)
+==================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Init command/response
+		Controller Index:	<controller id>
+		Command parameters:	Static Auth (16 octets)
+					Output Size (1 octet)
+					Output Actions (2 octets)
+					Input Size (1 octet)
+					Input Actions (2 octets)
+
+		This command is used to provide provisioning configuration for
+		node.
+
+		Output actions is a bitmask with following available bits:
+			1	Blink
+			2	Beep
+			3	Vibrate
+			4	Display number
+			5	Display string
+
+		Input actions is a bitmask with following available bits:
+			1	Push
+			2	Twist
+			3	Enter number
+			5	Enter string
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x03 - Reset command/response
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+
+		This command is used to reset state of a provisioned node making
+		it to start sending unprovisioned beacons.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x04 - Input number command/response
```

Comment:
You could merge "Input number command/response" and "Input string command/response" into one command and add action field, just like you did in "Input action Event"